README.md

```diff
@@ -22,14 +26,33 @@ not to show a table listing all the found file extensions and their respective
 frequencies, so that it will only display the total number of files.
 
 By default, it will ignore files and directories that are supposed to be
-hidden (with names starting with '.', but you can add the `-a` or `--all` optional
+hidden, but you can add the `-a` or `--all` optional
 switch argument to make it count all files.
 
+Hidden files and directories
+
+Windows: files and directories for which FILE_ATTRIBUTE_HIDDEN is true
+
+Linux, Mac OS: those with names starting with "."(dot)
+
 This utility can also be used to search for files that have a certain file extension
 (using `-fe` or `--file-extension`) and, optionally, display a short preview (`-p`or 
 `--preview`) for text files. The size of the preview text sample can optionally be
 customized by using the `-ps` or `--preview-size` argument followed by an integer number.
 
+The list of file types for which preview is available can be viewed with the `-st` or `--supported-types` argument.
+The names of extensions are case sensitive. The results for `ini` and `INI` will be different.
+
+By default, the result of a search by certain file extension is a list with the full paths of the files found.
```

Comment:
I have been thinking on making optional the file size display for each listed file path. It would allow to use this utility to build simple lists of paths separated by newlines that can be useful in some contexts. The adicional info (file size) may affect the ability for this list to be used by other tools.

Reply:
#51